Use base-files as well for Ubuntu since ubuntu-doc is not the greatest package to check.

From: Chuck Short <chuck.short@canonical.com>


---
 cobbler/action_import.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cobbler/action_import.py b/cobbler/action_import.py
index 6f2909c..c50b4d1 100644
--- a/cobbler/action_import.py
+++ b/cobbler/action_import.py
@@ -1294,7 +1294,7 @@ class UbuntuImporter ( DebianImporter ) :
   def get_release_files(self):
        if not self.get_pkgdir():
            return []
-       return glob.glob(os.path.join(self.get_pkgdir(), "main/u/ubuntu-docs" , "ubuntu-docs_*"))
+       return glob.glob(os.path.join(self.get_pkgdir(), "main/b/base-files" , "base-files_*"))
 
   def set_variance(self, flavor, major, minor, arch):
 
